### PR TITLE
Use jinja2

### PR DESCRIPTION
**Pull request by ashcrow**
- Typo/formatting fixes
- Logic simplification
- Uses `jinja2` for html output

### CODE REVIEW

File: utils/index.j2 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<html>
+  <head>
+    <title>Listing</title>
+  </head>
+<body>
+{% if files|length > 1 %}
+  {{ files|length }} entries<br><br>
+{% else %}
+  {{ files|length }} entry<br><br>
```
Comment (jlebon) [inline utils/index.j2]: Can we drop the two breaks here? Seems like the `<ul>` adds a nice spacing already!

**Comment by jlebon** [comment]
This is really cool! Fits the task at hand very nicely.

File: utils/index.j2 (hunk 1 above)
Comment (ashcrow) [reply utils/index.j2]: Yeah, looks like it would render fine without them now.

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by jlebon** [comment]
Sorry, it squashed it to one commit rather than two. Unfortunately we don't have the luxury of Homu yet to do it for us. :)

**Comment by jlebon** [comment]
Thanks for the contributions!

**Comment by ashcrow** [comment]
:smile:

**Pull request merged**